internal/local/dependencies_test.go

```diff
@@ -87,3 +87,48 @@ func TestDependenciesFind(t *testing.T) {
 		})
 	}
 }
+
+func TestDependenciesFindPackageJSON(t *testing.T) {
+	wd, wdErr := os.Getwd()
+	assert.Nil(t, wdErr)
+
+	testRoot := filepath.Join(wd, "testdata/dependencies")
+
+	t.Run("should return an empty data when run outside a project directory", func(t *testing.T) {
+		deps, err := FindPackageJSON(testRoot)
+		assert.Nil(t, err)
+		assert.Equal(t, Dependencies{}, deps)
+	})
+
+	t.Run("should return an error when a project has no package.json file", func(t *testing.T) {
+		dir := filepath.Join(testRoot, "empty")
+
+		_, err := FindPackageJSON(dir)
+		assert.Equal(t, fmt.Errorf("package.json not found at '%s/functions'", dir), err)
+	})
+
+	t.Run("should find a a package.json in directory format with an absolute path", func(t *testing.T) {
+		absPath, err := filepath.Abs("../local/testdata/dependencies/json")
```

Comment:
[optional] we could still use a test table (of size 2) similar to how the tests for `FindNodeModules` works

---

```suggestion
		absPath, err := filepath.Abs(filepath.Join(testRoot, "json"))
```